audit-log/README.md

```diff
@@ -0,0 +1,80 @@
+# Audit logging
+
+Audit logging på NAV gjøres pr i dag ved å sende CEF-formaterte meldinger til naudit som overfører de videre til ArcSight.
+
+CEF (Common Event Format) er et meldingsformat som brukes for å beskrive en hendelse. Og de ser f.eks slik ut:
+
+```
+CEF:0|my-application|AuditLogger|1.0|audit:access|NAV-ansatt har gjort oppslag på bruker|INFO|sproc=0bfe8d02-93d9-4530-99ac-3debae410769 duid=12345678900 end=1654185299215 suid=Z12345
+```
+
+## Hvordan ta i bruk audit logging
+
+1. Legg til avhengigheten for audit logging i **pom.xml** filen til applikasjonen:
+```xml
+<dependency>
+    <groupId>no.nav.common</groupId>
+    <artifactId>audit-log</artifactId>
+    <version>NEWEST_VERSION</version>
+</dependency>
+```
+
+2. Lag en logback fil (logback-naudit.xml) for å konfigurere opp audit loggeren som skal sende data til naudit.
```

Comment:
Denne er automatisk konfigurert om man bruker `logback-default.xml` fra `log` modulen. Usikker på om navnet på loggeren er lik, men det kunne vært kjekt om de funkerte sammen.

Reply:
Vi har egentlig gått vekk fra å bruke `logback-default.xml` siden den trekker inn ting uten at man får et forhold til det. Men er enig at det kunne vært greit med en deprecation kommentar i den gamle logback-naudit filen 👍

Reply:
Det er forsåvidt frivillig om man bruker `logback-default.xml`, manuelt inkluderer filene fra `log` modulen eller lager sine egne logback filer. 
Personlig er jeg fan av `logback-default` siden det hindrer at det skjer feil, e.g man logger til `SecureLog`  uten at `logback-securelogs.xml` er inkludert og dermed får sensitiv logg på feil sted. Men akkurat dette er ikke store greia for min del. 

Det jeg derimot ikke ønsker at vi går tilbake til copy-pasting av masse logback konfig mellom prosjekter. Så det jeg egentlig etterspørr en liten kommentar om at `logback-naudit.xml` eksisterer i `log` modulen. Så om man allerede bruker den modulen, så kan man bare inkludere `logback-naudit.xml` eller forsåvidt gjøre som beskrevet i doc'en og lage filen selv.

Reply:
La til kommentar 👍 Gikk også litt tilbake på dette med å kopiere inn naudit configen til appene som skal bruke det, slik at man inkluderer det slik som før, bortsett fra at man henter det fra  "audit_log" istedenfor "log"

Reply:
Helt supert det. :+1: 
På sikt kan vi da vurdere å slank `log` modulen for cxf/secure-logg oppsett. 🤞🏼